src/js/components/Menu/doc.js

```diff
@@ -139,6 +139,14 @@ export const themeDoc = {
     type: 'string',
     defaultValue: undefined,
   },
+  'menu.drop.align': {
+    description: 'The alignment of the drop with respect to the menu button.',
+    type: 'string',
```

Comment:
```suggestion
    type: 'string | object',
```
Like the default value suggests

Reply:
Good catch, thank you.

Reply:
Done.